### ISSUE TITLE
[$10][MSFT-192] [Member] - Low contrast for Learn More Button

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2858]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2.  Click on Web Extension tool WAVE for accessibility testing. 
3. Scroll down to Learn More and Click on contrast button. 
4. Now Observe the banner. 

**Expected Result**
Expected contrast ratio of 4:5:1.

**Actual Result**
Element has insufficient color contrast Ration 3:25:1 (foreground color: #FFFFFF, background color: ##278fff)

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=63cedfe6%2D52a5%2D4f33%2D9399%2D00e26b7dd4a2&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20%2D%20Learn%20More%20Contrast

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096265 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096265 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Dara-K** [comment]
Please use #006AD7 as the new blue color for the button.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096265 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by rashmi73** [comment]
@codeMinter @veshu @mishacucicea  unable to find page for it, can you please help in which repo it is?

**Comment by veshu** [comment]
@rashmi73 If you are searching for code then please check https://github.com/appirio-tech/accounts-app this code for Login/Registration related issues. Let us know if it doesn't help

**Comment by rashmi73** [comment]
@veshu as given in attachement, screen is not login/registration, it is actually home page, but i dont find any such screen in accounts app.

I already saw login/registration page there is no learn more button present in  it.

**Comment by veshu** [comment]
@rashmi73 Ok Please keep this as hold and pick another. I will get back to you later, whether if this page was on scope or not. You can keep this assigned to you and work on another, probably this might be not in scope.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2858 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2858 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2858 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097097 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097097 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Out of scope

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
